README.md

```diff
@@ -0,0 +1,11 @@
+# HLTB Mobile App
+
+![hltb Version][hltb-image]
+
+This is a open-source, unofficial, and non-commercial mobile app for [How Long To Beat](https://howlongtobeat.com).
+
+![](src/assets/img/hltb_icon.png)
+
+
```

Comment:
Since you recently cloned the project, can you please list the steps to build the project?

Reply:
Hi @nmoya. Yeah, I'm going to clone a fresh copy and re-build it in order to update README with every necessary step.

Reply:
Thank you so much!